manim/utils/hashing.py

```diff
@@ -74,7 +74,7 @@ def layer(func):
             if is_method:
                 return lambda self, obj: cls._handle_already_processed(
                     obj,
-                    default_function=lambda obj: func(self, obj),
```

Comment:
keep

Reply:
This change was suggested to my by pylint. I'm happy to change the naming back.